airgun/views/ldapauthentication.py

```diff
@@ -16,6 +17,12 @@ class LDAPAuthenticationsView(BaseLoggedInView):
         "//h1[text()='LDAP authentication sources' or "
         "text()='LDAP Authentication']"
     )
+    internal = AuthSourceAggregateCard(locator=".//h2[contains(text(), 'Internal')]/parent::div",
+                                       name="Internal")
```

Comment:
That's little unusual to provide both `locator` and `name`.

Since you subclass `AggregateStatusCard`, wouldn't it be better to override `ROOT` in subclass and provide only `name` here?
```python
internal = AuthSourceAggregateCard(name="Internal")


class AuthSourceAggregateCard(AggregateStatusCard):
    ROOT = './/h2[contains(text(), {@name|quote})]/parent::div'
```

Reply:
I did not know we can write like this. thanks for this.